### DISCUSSION TITLE
Не загружается вкладка с домофонами.

### DISCUSSION THREAD

**Discussion by Burdocknf** [rosteleset/SmartYard-Server#65] [Q&A]
Добрый день, установил сервер по инструкции, все вроде как запустилось. При попытке открыть вкладку с домофонами выдает ошибку 500. 
![image](https://github.com/rosteleset/SmartYard-Server/assets/169169282/563909fb-8df9-423a-82a6-0d63dd66dff3)
Помогите разобраться почему так происходит. Пробовал переустанавливать сервер несколько раз, все проверял но никак.

Так же при попытке перейти в настройки от админа выдает вот такую ошибку.
![image](https://github.com/rosteleset/SmartYard-Server/assets/169169282/a2f97b3d-4947-463c-8710-bfedac329296)

**Comment by preyai** [comment]
Добавьте в блок модулей "users".
Конфиг: `client/config/config.json`

**Reply by Burdocknf** [reply]
Заработало, спасибо.

**Reply by Burdocknf** [reply]
Теперь хотелось бы как-то с вкладкой с домофонами разобраться:)

**Comment by mmikel** [comment]
покажите что в логах (error) nginx'а...

**Reply by Burdocknf** [reply]
[error.log](https://github.com/rosteleset/SmartYard-Server/files/15237143/error.log)

**Comment by mmikel** [comment]
либо настройте zabbix, либо удалите секцию monitoring из конфига сервера

**Reply by Burdocknf** [reply]
Заработало, спасибо большое за помощь.
